src/Standards/AcquiaPHP/ruleset.xml

```diff
@@ -49,47 +51,6 @@
 
   <!-- PEAR sniffs -->
   <rule ref="PEAR.Files.IncludingFile"/>
-  <!-- Disable some error messages that we do not want. -->
-  <rule ref="PEAR.Files.IncludingFile.UseIncludeOnce">
-    <severity>0</severity>
-  </rule>
-  <rule ref="PEAR.Files.IncludingFile.UseInclude">
-    <severity>0</severity>
-  </rule>
-  <rule ref="PEAR.Files.IncludingFile.UseRequireOnce">
-    <severity>0</severity>
-  </rule>
-  <rule ref="PEAR.Files.IncludingFile.UseRequire">
-    <severity>0</severity>
-  </rule>
-  <rule ref="PEAR.Functions.FunctionCallSignature.OpeningIndent">
-    <severity>0</severity>
-  </rule>
-  <rule ref="PEAR.Functions.ValidDefaultValue"/>
-  <rule ref="PEAR.Functions.FunctionCallSignature"/>
-  <!-- The sniffs inside PEAR.Functions.FunctionCallSignature silenced below are
-    also silenced in Drupal CS' ruleset.xml. The code below is a 1-to-1 copy
-    from that file. -->
-  <!-- Disable some error messages that we already cover. -->
-  <rule ref="PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket">
-    <severity>0</severity>
-  </rule>
-  <rule ref="PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket">
-    <severity>0</severity>
-  </rule>
-  <!-- Disable some error messages that we do not want. -->
-  <rule ref="PEAR.Functions.FunctionCallSignature.Indent">
-    <severity>0</severity>
-  </rule>
-  <rule ref="PEAR.Functions.FunctionCallSignature.ContentAfterOpenBracket">
-    <severity>0</severity>
-  </rule>
-  <rule ref="PEAR.Functions.FunctionCallSignature.CloseBracketLine">
-    <severity>0</severity>
-  </rule>
-  <rule ref="PEAR.Functions.FunctionCallSignature.EmptyLine">
-    <severity>0</severity>
-  </rule>
```

Comment:
Just for my understanding, what's the reasoning behind this change? To be honest, I'm not certain why they were there in the first place other than that we copied them from Drupal core.